src/routes/+page.svelte

```diff
@@ -5,6 +5,8 @@
   /** @type {import('./$types').PageData} */
   export let data;
 
+  console.log({data});
+
   $: clothesCollection = data[0]?.node?.products?.edges;
   $: featuredCollection = data[1]?.node?.products?.edges;
```

Comment:
This will correspond to the return value of the `load` in `+page.server.js` — i.e. `data.products` (this should be enforced by TypeScript since the type of `data` is `import('./$types').PageData` — if that's not happening, there's a bug somewhere)